cmd/juju/status/status.go

```diff
@@ -134,6 +138,10 @@ Examples:
     # Provide output as valid JSON
     juju status --format=json
 
+    # Watch the status of the mysql application every five seconds
+	# Only available for unix-based systems.
+    juju status --watch 5s mysql
+
 Further reading:
 
     https://juju.is/docs/command/status
```

Comment:
A nit, but this URL doesn't exist

Reply:
Good point. I guess if there are more of these urls in the help for the other commands.

Reply:
Yeh, quite likely!